mu/modes/microbit.py

```diff
@@ -441,14 +441,14 @@ def get_hex_path(self):
             logger.error('Settings file {} could not be parsed.'.format(sp))
         else:
             if 'microbit_runtime_hex' in settings and \
-                    settings['microbit_runtime_hex'] is not None:
```

Comment:
Remove `is not None` so that with empty strings or lists we also return None directly instead of trying to figure out if they are runtime hex paths.